splinter/driver/webdriver/__init__.py

```diff
@@ -442,45 +442,43 @@ def get_alert(self, wait_time=None):
         except TimeoutException:
             return None
 
+    def _is_text_present(self, text):
+        """Check if text is present on the page.
+
+        NoSuchElementException is thrown if the body tag isn't present.
+        This has occasionally been observed. Assume that the page isn't fully
+        loaded yet.
+
+        StaleElementReferenceException is thrown if the page changes
+        quickly.
+        """
+        try:
+            self.driver.find_element_by_tag_name("body").text.index(text)
+            return True
+        except (NoSuchElementException, StaleElementReferenceException):
+            pass
+        except ValueError:
+            return False
```

Comment:
How the default value is `False` there is no need to separate `ValueError` from the another errors
```suggestion
        except (NoSuchElementException, StaleElementReferenceException, ValueError):
            pass
```